tests/data/format/max_line_length/function_docstrings.py

```diff
@@ -0,0 +1,13 @@
+def func():
```

Comment:
Basically the test should have 3 cases:
1) One multi-line summary clearly above 50 characters. > Summary should be wrapped.
2) One multi-line summary with a description above 50 chars. > Summary should be wrapped, but description shouldn't as we don't support that.
3) One single-line docstring that is 49 chars without the final quotes. See `LinesChunk`.
If it is a single-line docstring we need to add the final quotes on the same line, so the line is not actually 49 but 52, so we should indeed wrap. That is what `LinesChunck` is testing.

Reply:
OK. Updated them. But for some reason it seems that `50` is not picked up in this test.

Reply:
I think this is because the option doesn't use `type`. You could add a breakpoint somewhere after `line_length = self.config.max_line_length` but I think `line_length` will now be `"50"` instead of `50`. This can be fixed by using `type=int` in the `add_argument` call.

Reply:
It actually says that all docstrings are properly formatted. So it doesn't run